.gitignore

```diff
@@ -22,4 +22,6 @@ site/openstates
 +*.sublime*
 openstates.sqlite3
 dump/
-venv
\ No newline at end of file
+venv
+.idea/
```

Comment:
You might want to put these in your global github ignore and revert commits to this file -- otherwise they will be not so useful for the main branch or anyone not using Pycharm: https://robots.thoughtbot.com/global-gitignore